### DISCUSSION TITLE
svgEditor.setCustomHandlers is not a function

### DISCUSSION THREAD

**Discussion by 1001QAdotNET** [SVG-Edit/svgedit#783] [General]
I am reading this document https://github.com/SVG-Edit/svgedit/blob/master/docs/tutorials/Events.md
There is says that you could use the save method to customize the way editor works when you are saving the data

```
svgEditor.setCustomHandlers({
  save (_win, _data) {
    // Save svg
  }
})
```

Unfortunately this method is not available anymore svgEditor.setCustomHandlers or I might be missing something here

**Comment by 1001QAdotNET** [comment]
I do get it that I have to replace the existing module with this one here but it is still not working even if I do that
https://github.com/SVG-Edit/svgedit/blob/e10f70aec0c27f6d7ffbcddefe6a9400ab8782e9/archive/untested-extensions/extensions/ext-opensave/ext-opensave.js
Please if you have the cycles add this method back along with the existing ones it is useful when you want to use svg-edit as an external editor for another app

**Comment by jfhenon** [comment]
This is actually not a bug so I will move it to a discussion.
Thanks
JF

**Comment by jfhenon** [comment]
V7 has the same capability to replace the behavior to save, open, etc... 
You should look the file src/editor/extensions/ext-opensave/ext-opensave.js which implements the default behavior but you can customize as needed for your own application.
